<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="utf-8" />
	<link href="/bootstrap/css/bootstrap.css" rel="stylesheet" media="screen">
	<link href="/bootstrap/css/bootstrap-responsive.css" rel="stylesheet" media="screen">
	<link href="http://netdna.bootstrapcdn.com/font-awesome/3.1.1/css/font-awesome.css" rel="stylesheet">
   <script type="text/javascript" src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
	<link rel="stylesheet" href="/css/main.css" />
	<link rel="stylesheet" href="/css/d3.css" />
	<link rel="stylesheet" href="/css/pygments.css" />
    <script src="http://d3js.org/d3.v3.min.js"></script>
    <link rel="alternate" type="application/atom+xml" title="RSS Feed for Tim's blog" href="http://feeds.feedburner.com/timpoisot" />
    
    <script type="text/javascript">
		var _gaq = _gaq || [];
		_gaq.push(['_setAccount', 'UA-11830706-1']);
		_gaq.push(['_trackPageview']);

		(function() {
		 var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
		 ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
		 var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
		})();
	</script>
	
	<meta name="author" content="Tim Poisot" />
   <title>Tim Poisot :: Blog</title>
    
</head>

<body>

<div class="header sitewide">
<div class="container">
<div class="row-fluid narrow">
   <div class="span10 offset1 main-nav"><h1>Tim Poisot</h1><nav><ul class="nav nav-pills">
  <li><a href="/">Home</a></li>
  <li><a href="/papers/">Publications</a></li>
  <li><a href="/research/">Research</a></li>
  <li class="active"><a href="/blog/">Blog</a></li>
</ul></nav>

</div>
</div>
<!-- <div class="row-fluid site-head">
	<div class="span10 offset1">
	<h1>Tim Poisot</h1> <h2>Community ecologist & open scientist</h2>

	</div>
</div> -->
</div>
</div>

<div class="container main">
	<div class="row-fluid narrow">
		
		<div class="span10 offset1 page-content">
		
<div class='post'>
	<h1 class='post-title'><span class="label label-info">Preprint</span> Measuring realized network modularity</h1>
	<div class='postinfo'>May 23, 2013</div>
	<p>A paper describing how to measure realized network modularity just appeared online at F1000 Research. It's currently waiting for reviewers. This is a short methodological note, to "solve" a problem I had when presenting the ouput of modularity analyses. Most of the time, the values you get are between 0 and 1, but they are fairly difficult to make sense of. They're not probabilities that the network is different. They're statistics to some extent, but it's not clear what a...</p>
   <a href="/2013/05/23/realized-modularity/" class='keeponreading'>Keep on reading!</a></p>
	<div class='postinfo'>
		
			<span class="label label-default">ecological networks</span>
		
	</div>
</div>

<div class='post'>
	<h1 class='post-title'><span class="label label-success">Paper</span> Trophic complementarity and the BEF relationship in food webs</h1>
	<div class='postinfo'>May 22, 2013</div>
	<p>I'm really glad to announce that we (here meaning Dom Gravel, Nicolas Mouquet and I), have a new paper in Ecology Letters, about the biodiversity-ecosystem functioning (BEF) relationship in food webs. In this paper, we show that trophic complementarity is a very good predictor of productivity in complex ecosystems. You can go get the paper. So what is trophic complementarity, and why is this result important? The strength of the BEF theory is that the core mechanisms are intuitive and...</p>
   <a href="/2013/05/22/food-web-structure-ecosystem-functioning/" class='keeponreading'>Keep on reading!</a></p>
	<div class='postinfo'>
		
			<span class="label label-default">ecological networks</span>
		
			<span class="label label-default">ecosystem functioning</span>
		
			<span class="label label-default">ecological modeling</span>
		
	</div>
</div>

<div class='post'>
	<h1 class='post-title'><span class="label label-warning">Essay</span> Should code be reviewed?</h1>
	<div class='postinfo'>May 21, 2013</div>
	<p>Joppa and colleagues published a policy forum in Science, dealing with problems in the use of adoption of scientific software. Their paper quite nicely summarizes that the intrinsic quality of a software is not a deciding factor in its use, and that the community of scientists will be more influenced by who the early adopters are, and how well they are connected. In short, using a particular software quickly becomes a tradition, with few considerations for its merit, and worse,...</p>
   <a href="/2013/05/21/should-code-reviewed/" class='keeponreading'>Keep on reading!</a></p>
	<div class='postinfo'>
		
			<span class="label label-default">open source</span>
		
			<span class="label label-default">open science</span>
		
			<span class="label label-default">peer review</span>
		
			<span class="label label-default">software</span>
		
			<span class="label label-default">next-gen science</span>
		
	</div>
</div>

<div class='post'>
	<h1 class='post-title'>Remove indels from blast output</h1>
	<div class='postinfo'>May 20, 2013</div>
	<p>My wife recently mentionned to me that most of the errors introduced during 454 pyrosequencing were indels rather than point mutations. It may not sound like it, but it's kind of a big deal when measuring similarity between sequences, most likely when you try to do taxonomic identification by comparing to the best match of blast. So to get a "reliable" estimate of the dissimilarity, a reliable strategy is most likely to remove the gaps. As doing it by hand...</p>
   <a href="/2013/05/20/blast-remove-gaps/" class='keeponreading'>Keep on reading!</a></p>
	<div class='postinfo'>
		
			<span class="label label-default">open lab book</span>
		
			<span class="label label-default">bioinformatics</span>
		
			<span class="label label-default">python</span>
		
			<span class="label label-default">biopython</span>
		
			<span class="label label-default">pyrosequencing</span>
		
	</div>
</div>

<div class='post'>
	<h1 class='post-title'>Using make and pandoc for reproducible papers</h1>
	<div class='postinfo'>May 18, 2013</div>
	<p>I tend to use pandoc more and more to write my papers. The syntax of markdown is ligthweight, doesn't get in the way of my writing, and it's easy to export the text to almost any format you can name afterward. The only issue I ran in is that I also like pgfplots a lot for graphics, because they come with the ability to update themselves at each compilation, when the data are re-generated. It's quite important if you want...</p>
   <a href="/2013/05/18/make-pandoc/" class='keeponreading'>Keep on reading!</a></p>
	<div class='postinfo'>
		
			<span class="label label-default">open lab book</span>
		
			<span class="label label-default">pandoc</span>
		
	</div>
</div>

<div class='post'>
	<h1 class='post-title'>Steal my paper! (seriously, please, don't)</h1>
	<div class='postinfo'>May 16, 2013</div>
	<p>With Phil's paper on preprints out, and the idea that writing manuscripts in the open introduced by Karthik Ram, it's almost not completely crazy to write research papers in an entirely open space, like github or Google Docs. I'm currently doing it (everything with paper or ms in the title of the repository is something not yet published), and so are a few other people. And it's a good thing. It might actually be the better way to disseminate research...</p>
   <a href="/2013/05/16/steal-my-paper/" class='keeponreading'>Keep on reading!</a></p>
	<div class='postinfo'>
		
			<span class="label label-default">open science</span>
		
			<span class="label label-default">data sharing</span>
		
	</div>
</div>

<div class='post'>
	<h1 class='post-title'>A C version of the Nicholson-Bailey model</h1>
	<div class='postinfo'>May  6, 2013</div>
	<p>The Nicholson-Bailey model is a well known model of host and parasitoid population dynamics. It assumes that parasitoids will search a fraction of their habitat for hosts, and then kill them. Yes, it's brutal like that. There are a few fun things with this model. It's light in parameters (the single-patch version requires only three parameters), it's in discrete time (so it's easy to program), and it's classical put on a lattice (so it's a good way to program these...</p>
   <a href="/2013/05/06/nicholson-bailey-c/" class='keeponreading'>Keep on reading!</a></p>
	<div class='postinfo'>
		
			<span class="label label-default">open lab book</span>
		
			<span class="label label-default">C</span>
		
			<span class="label label-default">parasites</span>
		
			<span class="label label-default">models</span>
		
			<span class="label label-default">species interactions</span>
		
	</div>
</div>

<div class='post'>
	<h1 class='post-title'><span class="label label-info">Preprint</span> Data sharing and preprints in ecology</h1>
	<div class='postinfo'>April 30, 2013</div>
	<p>I'm glad to announce that we (I, Ross Mounce, and Dom Gravel), have put up a preprint about data sharing in ecology on figshare. The manuscript is under consideration at Ideas in Ecology &amp; Evolution. We try to make a valid point about why ecologists should share their data, how we can do it in practice, and the role that funding agencies and journals can play. And speaking of preprints, but also sure to check out Philippe's opinion piece on...</p>
   <a href="/2013/04/30/data-sharing-ecology/" class='keeponreading'>Keep on reading!</a></p>
	<div class='postinfo'>
		
			<span class="label label-default">open science</span>
		
			<span class="label label-default">data sharing</span>
		
	</div>
</div>

<div class='post'>
	<h1 class='post-title'><span class="label label-warning">Essay</span> Learning how to code (in ecology)</h1>
	<div class='postinfo'>April 12, 2013</div>
	<p>A few days ago, I received an email asking about pointers on how to program in ecology. I'm not sure I'm the best person to ask, but writing the answer, I realized that there might be potential to share tips with other people. Now, a little bit of foreword. I come from a very empirical background. My undergrad studies are in genetics and immunology, and up until the end of my Masters, I have been spending time either in the...</p>
   <a href="/2013/04/12/learn-code-ecology/" class='keeponreading'>Keep on reading!</a></p>
	<div class='postinfo'>
		
			<span class="label label-default">programing</span>
		
			<span class="label label-default">software</span>
		
	</div>
</div>

<div class='post'>
	<h1 class='post-title'>Elsevier acquired Mendeley. It's not as bad as I thought.</h1>
	<div class='postinfo'>April  9, 2013</div>
	<p>Mendeley was recently acquired by public enemy Elsevier in a puzzling move. Some of you may remember that I've announced that, should this acquisition happen, I'll be out of here. Actually, I've switched to Zotero a while back and never ever looked back. But still. Mr. Will Gunn himself asked me to do a follow-up post, on the basis that it's not as bad as I feared. And as much as I am pained to admit it, that is entirely...</p>
   <a href="/2013/04/09/mendelsevier/" class='keeponreading'>Keep on reading!</a></p>
	<div class='postinfo'>
		
			<span class="label label-default">open science</span>
		
	</div>
</div>

<div class='post'>
	<h1 class='post-title'>Is E.O. Wilson wrong about maths?</h1>
	<div class='postinfo'>April  7, 2013</div>
	<p>E.O. Wilson recently published an editorial entitled "Great scientists don't need math" in the Wall Street Journal. Jeremy Fox wrote a great rebuttal of it, which you should certainly read as well. In short, Wilson's argument is that most of great science is the work of intuition. In Jean Cocteau's words, science is the confirmation of instinct's discoveries. And in this perspective, math literacy is not needed, because it's only a boring formality, to confirm what the "great minds" have...</p>
   <a href="/2013/04/07/wilson-maths/" class='keeponreading'>Keep on reading!</a></p>
	<div class='postinfo'>
		
			<span class="label label-default">ecological theory</span>
		
			<span class="label label-default">teaching</span>
		
	</div>
</div>

<div class='post'>
	<h1 class='post-title'><span class="label label-success">Paper</span> High-throughput sequencing and community ecology</h1>
	<div class='postinfo'>March 12, 2013</div>
	<p>High-throughput sequencing (let's abbreviate it as HTS), the ability to generate millions of sequences for a relatively low cost and effort, can make a huge difference in community ecology and biogeography. Think about it this way: for a few days of work, you can get a really exhaustive sampling of the diversity, with measures of relative abundance. I'm currently using 454 pyrosequencing in a project, and when I started reviewing the literature, I was stricken by the fact that there...</p>
   <a href="/2013/03/12/ngs-community-ecology/" class='keeponreading'>Keep on reading!</a></p>
	<div class='postinfo'>
		
			<span class="label label-default">biogeography</span>
		
			<span class="label label-default">next-generation sequencing</span>
		
			<span class="label label-default">ecology</span>
		
	</div>
</div>

<div class='post'>
	<h1 class='post-title'>A d3.js alternative to stacked bar charts</h1>
	<div class='postinfo'>February 18, 2013</div>
	<p>Following yesterday's post on an alternative to barcharts, I decided to write an implementation of this graph in D3.js. The more I play around with it, the more I like it. As it turned out, it was fairly easy, so I'll share the code as a gist. For this particular example, I assume that the data are in a matrix, in csv format, with sites as the first row, and species abundances in columns. I also assume that the abudances...</p>
   <a href="/2013/02/18/stacked-barcharts-d3/" class='keeponreading'>Keep on reading!</a></p>
	<div class='postinfo'>
		
			<span class="label label-default">data visualization</span>
		
			<span class="label label-default">D3.js</span>
		
	</div>
</div>

<div class='post'>
	<h1 class='post-title'>An alternative to stacked bar charts</h1>
	<div class='postinfo'>February 17, 2013</div>
	<p>I hate stacked bar charts. I see them in so many papers, using the awful Excel default options, and I hate them. I feel them visually unappealing, confusing, and extremely inefficient at displaying information. It's almost impossible to compare the size of two bars without measuring them, which means that they carry less information than a good old table. So I was looking for alternatives to them. Here is the code I ended up with. First let's load a few...</p>
   <a href="/2013/02/17/stacked-barcharts/" class='keeponreading'>Keep on reading!</a></p>
	<div class='postinfo'>
		
			<span class="label label-default">data visualization</span>
		
	</div>
</div>

<div class='post'>
	<h1 class='post-title'>Estimating edges contribution to network properties</h1>
	<div class='postinfo'>February 13, 2013</div>
	<p>I had so much fun writing the previous post about how it is possible to measure nodes contribution to network properties that I wanted to see what happens when you remove edges. Also, I spent two hours this afternoon playing with D3.js, and I wanted to put what I've learnt in practice. What I want to do is remove each edge, and see how it affects modularity. This is, once again, really easy to do in networkx. We'll just write...</p>
   <a href="/2013/02/13/edges-contribution-modularity/" class='keeponreading'>Keep on reading!</a></p>
	<div class='postinfo'>
		
			<span class="label label-default">ecological networks</span>
		
			<span class="label label-default">modularity</span>
		
			<span class="label label-default">open lab book</span>
		
			<span class="label label-default">python</span>
		
			<span class="label label-default">networkx</span>
		
			<span class="label label-default">D3.js</span>
		
	</div>
</div>


<!-- Pagination links -->
<div id="post-pagination" class="pagination">
 
  <ul class="pages">
    <li class="page">
      
        <a href="/blog/">1</a>
      
    </li>

    
      <li class="page">
        
          <span class="current-page">2</span>
        
      </li>
    
      <li class="page">
        
          <a href="/blog/page3">3</a>
        
      </li>
    
      <li class="page">
        
          <a href="/blog/page4">4</a>
        
      </li>
    
      <li class="page">
        
          <a href="/blog/page5">5</a>
        
      </li>
    
  </ul>

</div>


		</div>
	
	</div>
</div>

<div class="container footer">
   <div class="row-fluid narrow">

      <div class="span10 offset1 footer-content">
         	  <p class="muted credit">
	  This website uses <a href='http://twitter.github.io/bootstrap/index.html'>bootstrap</a>, <a href='http://fortawesome.github.io/Font-Awesome/'>Font Awesome</a>, is built with <a href='https://github.com/mojombo/jekyll'>Jekyll</a>, and hosted by <a href='http://pages.github.com/'>GitHub pages</a>. Contents under the <a href='http://creativecommons.org/licenses/by-sa/3.0/deed.en_US'>CC BY-SA 3.0</a> licence.
     </p>	 
<p class="lead" style="text-align: center;">
<a href="mailto:timothee_poisot@uqar.ca"><i class="icon-fixed-width icon-envelope"></i></a>&nbsp;
<a href="http://feeds.feedburner.com/timpoisot"><i class="icon-fixed-width icon-rss"></i></a>&nbsp;
<a href="https://github.com/tpoisot/"><i class="icon-fixed-width icon-github"></i></a>&nbsp;
<a href="https://twitter.com/tpoi/"><i class="icon-fixed-width icon-twitter"></i></a>
</p>

      </div>

   </div>
</div>

</body>

</html>
