<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="utf-8" />
   <link rel="stylesheet" href="/css/normalize.css">
	<link rel="stylesheet" href="/css/foundation.min.css">
	<link href="http://netdna.bootstrapcdn.com/font-awesome/3.1.1/css/font-awesome.css" rel="stylesheet">
   <script type="text/javascript" src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
	<link rel="stylesheet" href="/css/d3.css" />
	<link rel="stylesheet" href="/css/pygments.css" />
   <link rel="stylesheet/less" type="text/css" href="/css/tim.less" />
   <script type="text/javascript" src="/lib/less.js"></script>
   <script src="http://d3js.org/d3.v3.min.js"></script>
   <link rel="alternate" type="application/atom+xml" title="RSS Feed for Tim's blog" href="http://feeds.feedburner.com/timpoisot" />
    
    <script type="text/javascript">
		var _gaq = _gaq || [];
		_gaq.push(['_setAccount', 'UA-11830706-1']);
		_gaq.push(['_trackPageview']);

		(function() {
		 var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
		 ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
		 var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
		})();
	</script>
	
	<meta name="author" content="Tim Poisot" />
   <title>Tim Poisot :: Blog</title>
    
</head>

<body>

<div id="main">

<div class="row navigation">
   <div class="large-10 large-offset-1"><ul>
  <li><a href="/">Home</a></li>
  <li><a href="/papers/">Publications</a></li>
  <li><a href="/research/">Research</a></li>
  <li class="active"><a href="/blog/">Blog</a></li>
</ul>
</div>
</div>

<div class="row header">
   <div class="large-12"><h1>Tim Poisot</h1>
<h2>Spatial ecology of species interactions</h2>
</div>
</div>

<div class="row content">
   <div class="large-12 columns">
   <!-- This loops through the paginated posts -->

<div class='post'>
  <h1>The structure of natural bacteria-phage networks</h1>
  <div class="postinfo">May 30, 2013 &nbsp;&nbsp;&nbsp;&nbsp; <span class="label label-default" style="margin-right: 3px;">ecological networks</span><span class="label label-default" style="margin-right: 3px;">bacteria</span><span class="label label-default" style="margin-right: 3px;">bacteriophages</span></div>
  <p>
    Freedom! The final chapter of my thesis is now online on the website of Ecological Processes. We use data form a field isolated bacteria-phage system to have a network approach of the specialist-generalist coexistence problem. The main result is that, even when controlling for phylogenetic dispersal of hosts, it's better to be a generalist. This signal is highly consistent across the five networks we studied, and goes in the way of recent results on coevolution: if you let a system...
  </p>
<a class="readmore" href="/2013/05/30/bacteria-soil-networks/">Read more...</a>
</div>

<div class='post'>
  <h1>Measuring realized network modularity</h1>
  <div class="postinfo">May 23, 2013 &nbsp;&nbsp;&nbsp;&nbsp; <span class="label label-default" style="margin-right: 3px;">ecological networks</span></div>
  <p>
    A paper describing how to measure realized network modularity just appeared online at F1000 Research. It's currently waiting for reviewers. This is a short methodological note, to "solve" a problem I had when presenting the ouput of modularity analyses. Most of the time, the values you get are between 0 and 1, but they are fairly difficult to make sense of. They're not probabilities that the network is different. They're statistics to some extent, but it's not clear what a...
  </p>
<a class="readmore" href="/2013/05/23/realized-modularity/">Read more...</a>
</div>

<div class='post'>
  <h1>Trophic complementarity and the BEF relationship in food webs</h1>
  <div class="postinfo">May 22, 2013 &nbsp;&nbsp;&nbsp;&nbsp; <span class="label label-default" style="margin-right: 3px;">ecological networks</span><span class="label label-default" style="margin-right: 3px;">ecosystem functioning</span><span class="label label-default" style="margin-right: 3px;">ecological modeling</span></div>
  <p>
    I'm really glad to announce that we (here meaning Dom Gravel, Nicolas Mouquet and I), have a new paper in Ecology Letters, about the biodiversity-ecosystem functioning (BEF) relationship in food webs. In this paper, we show that trophic complementarity is a very good predictor of productivity in complex ecosystems. You can go get the paper. So what is trophic complementarity, and why is this result important? The strength of the BEF theory is that the core mechanisms are intuitive and...
  </p>
<a class="readmore" href="/2013/05/22/food-web-structure-ecosystem-functioning/">Read more...</a>
</div>

<div class='post'>
  <h1>Should code be reviewed?</h1>
  <div class="postinfo">May 21, 2013 &nbsp;&nbsp;&nbsp;&nbsp; <span class="label label-default" style="margin-right: 3px;">open source</span><span class="label label-default" style="margin-right: 3px;">open science</span><span class="label label-default" style="margin-right: 3px;">peer review</span><span class="label label-default" style="margin-right: 3px;">software</span><span class="label label-default" style="margin-right: 3px;">next-gen science</span></div>
  <p>
    Joppa and colleagues published a policy forum in Science, dealing with problems in the use of adoption of scientific software. Their paper quite nicely summarizes that the intrinsic quality of a software is not a deciding factor in its use, and that the community of scientists will be more influenced by who the early adopters are, and how well they are connected. In short, using a particular software quickly becomes a tradition, with few considerations for its merit, and worse,...
  </p>
<a class="readmore" href="/2013/05/21/should-code-reviewed/">Read more...</a>
</div>

<div class='post'>
  <h1>Remove indels from blast output</h1>
  <div class="postinfo">May 20, 2013 &nbsp;&nbsp;&nbsp;&nbsp; <span class="label label-default" style="margin-right: 3px;">open lab book</span><span class="label label-default" style="margin-right: 3px;">bioinformatics</span><span class="label label-default" style="margin-right: 3px;">python</span><span class="label label-default" style="margin-right: 3px;">biopython</span><span class="label label-default" style="margin-right: 3px;">pyrosequencing</span></div>
  <p>
    My wife recently mentionned to me that most of the errors introduced during 454 pyrosequencing were indels rather than point mutations. It may not sound like it, but it's kind of a big deal when measuring similarity between sequences, most likely when you try to do taxonomic identification by comparing to the best match of blast. So to get a "reliable" estimate of the dissimilarity, a reliable strategy is most likely to remove the gaps. As doing it by hand...
  </p>
<a class="readmore" href="/2013/05/20/blast-remove-gaps/">Read more...</a>
</div>


<div class="pagination-centered">
  
  <ul class="pagination">
     
     <li><a href="/blog/">1</a></li>
     

    
    
    <li><a href="/blog/page2">2</a></li>
    
    
    
    <li><a href="/blog/page3">3</a></li>
    
    
    
    <li><a href="/blog/page4">4</a></li>
    
    
    
    <li class="current">5</li>
    
    
    
    <li><a href="/blog/page6">6</a></li>
    
    
    
    <li><a href="/blog/page7">7</a></li>
    
    
    
    <li><a href="/blog/page8">8</a></li>
    
    
    
    <li><a href="/blog/page9">9</a></li>
    
    
    
    <li><a href="/blog/page10">10</a></li>
    
    
    
    <li><a href="/blog/page11">11</a></li>
    
    
    
    <li><a href="/blog/page12">12</a></li>
    
    
    
    <li><a href="/blog/page13">13</a></li>
    
    
    
    <li><a href="/blog/page14">14</a></li>
    
    
    
    <li><a href="/blog/page15">15</a></li>
    
    
    
    <li><a href="/blog/page16">16</a></li>
    
    
  </ul>

</div>


   </div>
</div>

<div class="row footer">
   <div class="large-12 columns">
      	  <p class="muted credit">
	  This website uses <a href='http://http://foundation.zurb.com/'>Foundation 4</a>, <a href='http://fortawesome.github.io/Font-Awesome/'>Font Awesome</a>, is built with <a href='https://github.com/mojombo/jekyll'>Jekyll</a>, and hosted by <a href='http://pages.github.com/'>GitHub pages</a>. Contents under the <a href='http://creativecommons.org/licenses/by-sa/3.0/deed.en_US'>CC BY-SA 3.0</a> licence.
     </p>	 
<p class="lead" style="text-align: center;">
<a href="mailto:timothee_poisot@uqar.ca"><i class="icon-fixed-width icon-envelope"></i></a>&nbsp;
<a href="http://feeds.feedburner.com/timpoisot"><i class="icon-fixed-width icon-rss"></i></a>&nbsp;
<a href="https://github.com/tpoisot/"><i class="icon-fixed-width icon-github"></i></a>&nbsp;
<a href="https://twitter.com/tpoi/"><i class="icon-fixed-width icon-twitter"></i></a>
</p>

   </div>
</div>

</div>

</body>

</html>
