<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="utf-8" />
   <link rel="stylesheet" href="/css/normalize.css">
	<link rel="stylesheet" href="/css/foundation.min.css">
   <link href="//netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.css" rel="stylesheet">
   <script type="text/javascript" src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
	<link rel="stylesheet" href="/css/d3.css" />
	<link rel="stylesheet" href="/css/pygments.css" />
   <link rel="stylesheet/less" type="text/css" href="/css/tim.less" />
   <script type="text/javascript" src="/lib/less.js"></script>
   <link rel="stylesheet" href="/lib/highlight/styles/tomorrow.css" />
   <script type="text/javascript" src="/lib/highlight/highlight.pack.js"></script>
   <script>hljs.initHighlightingOnLoad();</script>
   <script src="http://d3js.org/d3.v3.min.js"></script>
   <link rel="alternate" type="application/atom+xml" title="RSS Feed for Tim's blog" href="http://feeds.feedburner.com/timpoisot" />
    
    <script type="text/javascript">
		var _gaq = _gaq || [];
		_gaq.push(['_setAccount', 'UA-11830706-1']);
		_gaq.push(['_trackPageview']);

		(function() {
		 var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
		 ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
		 var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
		})();
	</script>
	
	<meta name="author" content="Tim Poisot" />
   <title>Timothée Poisot :: Blog</title>
    
</head>

<body>
<a id="top"></a>

            <div class='contain-to-grid fixed'>
<nav class="top-bar" data-topbar>
     <ul class="title-area">
       <li class="name">
       <h1><a href="#top">Timothée Poisot</a></h1>
       </li>
       <li class="toggle-topbar menu-icon"><a href="#">Menu</a></li>
     </ul>

       <section class="top-bar-section">
         <!-- Right Nav Section -->
            <ul class="no-bullet main-nav right">
               <li><a href="/">Home</a></li>
               <li><a href="/papers/">Papers</a></li>
               <li><a href="/software/">Software</a></li>
               <li><a href="/research/">Research</a></li>
               <li class="active"><a href="/blog/">Blog</a></li>
            </ul>

             <!-- Left Nav Section -->
             <ul class="left">
                <li><a>Spatial ecology of species interactions</a></li>
             </ul>
           </section>
        </nav>
     </div>

<div class="row wrapper">
   <div class = "copy">
      <div class="large-12 small-12 columns maintext">
         <!-- This loops through the paginated posts -->


<div class='post postlist'>
   <h1>Trophic complementarity and the BEF relationship in food webs</h1>
   <div class="postinfo">
      May 22, 2013 &mdash; <span class="label radius small secondary" style="margin-right: 3px;"><a href="/tag/ecological networks/">ecological networks</a></span><span class="label radius small secondary" style="margin-right: 3px;"><a href="/tag/ecosystem functioning/">ecosystem functioning</a></span><span class="label radius small secondary" style="margin-right: 3px;"><a href="/tag/ecological modeling/">ecological modeling</a></span>
   </div>
      <p>I’m really glad to announce that we (here meaning Dom Gravel, Nicolas Mouquet and I), have a new paper in Ecology Letters, about the biodiversity-ecosystem functioning (BEF) relationship in food webs. In this paper, we show that trophic complementarity is a very good predictor of productivity in complex ecosystems. You can go get the paper. So what is trophic complementarity, and why is this result important? The strength of the BEF theory is that the...</p>
      <a class="readmore" href="/2013/05/22/food-web-structure-ecosystem-functioning/">Read more...</a>
   <hr />
</div>

<div class='post postlist'>
   <h1>Should code be reviewed?</h1>
   <div class="postinfo">
      May 21, 2013 &mdash; <span class="label radius small secondary" style="margin-right: 3px;"><a href="/tag/open source/">open source</a></span><span class="label radius small secondary" style="margin-right: 3px;"><a href="/tag/open science/">open science</a></span><span class="label radius small secondary" style="margin-right: 3px;"><a href="/tag/peer review/">peer review</a></span><span class="label radius small secondary" style="margin-right: 3px;"><a href="/tag/software/">software</a></span><span class="label radius small secondary" style="margin-right: 3px;"><a href="/tag/next-gen science/">next-gen science</a></span>
   </div>
      <p>Joppa and colleagues published a policy forum in Science, dealing with problems in the use of adoption of scientific software. Their paper quite nicely summarizes that the intrinsic quality of a software is not a deciding factor in its use, and that the community of scientists will be more influenced by who the early adopters are, and how well they are connected. In short, using a particular software quickly becomes a tradition, with few considerations...</p>
      <a class="readmore" href="/2013/05/21/should-code-reviewed/">Read more...</a>
   <hr />
</div>

<div class='post postlist'>
   <h1>Remove indels from blast output</h1>
   <div class="postinfo">
      May 20, 2013 &mdash; <span class="label radius small secondary" style="margin-right: 3px;"><a href="/tag/open lab book/">open lab book</a></span><span class="label radius small secondary" style="margin-right: 3px;"><a href="/tag/bioinformatics/">bioinformatics</a></span><span class="label radius small secondary" style="margin-right: 3px;"><a href="/tag/python/">python</a></span><span class="label radius small secondary" style="margin-right: 3px;"><a href="/tag/biopython/">biopython</a></span><span class="label radius small secondary" style="margin-right: 3px;"><a href="/tag/pyrosequencing/">pyrosequencing</a></span>
   </div>
      <p>My wife recently mentionned to me that most of the errors introduced during 454 pyrosequencing were indels rather than point mutations. It may not sound like it, but it’s kind of a big deal when measuring similarity between sequences, most likely when you try to do taxonomic identification by comparing to the best match of blast. So to get a “reliable” estimate of the dissimilarity, a reliable strategy is most likely to remove the gaps....</p>
      <a class="readmore" href="/2013/05/20/blast-remove-gaps/">Read more...</a>
   <hr />
</div>

<div class='post postlist'>
   <h1>Using make and pandoc for reproducible papers</h1>
   <div class="postinfo">
      May 18, 2013 &mdash; <span class="label radius small secondary" style="margin-right: 3px;"><a href="/tag/open lab book/">open lab book</a></span><span class="label radius small secondary" style="margin-right: 3px;"><a href="/tag/pandoc/">pandoc</a></span>
   </div>
      <p>I tend to use pandoc more and more to write my papers. The syntax of markdown is ligthweight, doesn’t get in the way of my writing, and it’s easy to export the text to almost any format you can name afterward. The only issue I ran in is that I also like pgfplots a lot for graphics, because they come with the ability to update themselves at each compilation, when the data are re-generated. It’s...</p>
      <a class="readmore" href="/2013/05/18/make-pandoc/">Read more...</a>
   <hr />
</div>

<div class='post postlist'>
   <h1>Steal my paper! (seriously, please, don't)</h1>
   <div class="postinfo">
      May 16, 2013 &mdash; <span class="label radius small secondary" style="margin-right: 3px;"><a href="/tag/open science/">open science</a></span><span class="label radius small secondary" style="margin-right: 3px;"><a href="/tag/data sharing/">data sharing</a></span>
   </div>
      <p>With Phil’s paper on preprints out, and the idea that writing manuscripts in the open introduced by Karthik Ram, it’s almost not completely crazy to write research papers in an entirely open space, like github or Google Docs. I’m currently doing it (everything with paper or ms in the title of the repository is something not yet published), and so are a few other people. And it’s a good thing. It might actually be the...</p>
      <a class="readmore" href="/2013/05/16/steal-my-paper/">Read more...</a>
   <hr />
</div>

<div class='post postlist'>
   <h1>A C version of the Nicholson-Bailey model</h1>
   <div class="postinfo">
      May  6, 2013 &mdash; <span class="label radius small secondary" style="margin-right: 3px;"><a href="/tag/open lab book/">open lab book</a></span><span class="label radius small secondary" style="margin-right: 3px;"><a href="/tag/C/">C</a></span><span class="label radius small secondary" style="margin-right: 3px;"><a href="/tag/parasites/">parasites</a></span><span class="label radius small secondary" style="margin-right: 3px;"><a href="/tag/models/">models</a></span><span class="label radius small secondary" style="margin-right: 3px;"><a href="/tag/species interactions/">species interactions</a></span>
   </div>
      <p>The Nicholson-Bailey model is a well known model of host and parasitoid population dynamics. It assumes that parasitoids will search a fraction of their habitat for hosts, and then kill them. Yes, it’s brutal like that. There are a few fun things with this model. It’s light in parameters (the single-patch version requires only three parameters), it’s in discrete time (so it’s easy to program), and it’s classical put on a lattice (so it’s a...</p>
      <a class="readmore" href="/2013/05/06/nicholson-bailey-c/">Read more...</a>
   <hr />
</div>


<div class="pagination-centered">
  
  <ul class="pagination">
     
     <li><a href="/blog/">1</a></li>
     

    
    
    <li><a href="/blog/page2">2</a></li>
    
    
    
    <li><a href="/blog/page3">3</a></li>
    
    
    
    <li><a href="/blog/page4">4</a></li>
    
    
    
    <li><a href="/blog/page5">5</a></li>
    
    
    
    <li><a href="/blog/page6">6</a></li>
    
    
    
    <li><a href="/blog/page7">7</a></li>
    
    
    
    <li class="current">8</li>
    
    
    
    <li><a href="/blog/page9">9</a></li>
    
    
    
    <li><a href="/blog/page10">10</a></li>
    
    
    
    <li><a href="/blog/page11">11</a></li>
    
    
    
    <li><a href="/blog/page12">12</a></li>
    
    
    
    <li><a href="/blog/page13">13</a></li>
    
    
    
    <li><a href="/blog/page14">14</a></li>
    
    
    
    <li><a href="/blog/page15">15</a></li>
    
    
    
    <li><a href="/blog/page16">16</a></li>
    
    
    
    <li><a href="/blog/page17">17</a></li>
    
    
  </ul>

</div>


      </div>
   </div>
   <div class='push'></div>
</div>

<div class="row footer">
   <div class="footercontent">
      <div class="small-4 columns">
<h5>Contact</h5>
<ul class='no-bullet'>
   <li><i class="fa fa-fw fa-envelope"></i> <a href="mailto:t.poisot@gmail.com">E-mail</a></li>
   <li><i class="fa fa-fw fa-twitter-square"></i> <code>@tpoi</code> on <a href="http://twitter.com/tpoi/">Twitter</a></li>
   <li><i class="fa fa-fw fa-github-square"></i> <code>tpoisot</code> on <a href="http://github.com/tpoisot/">GitHub</a></li>
   <li><i class="fa fa-fw fa-rss-square"></i> <a href="http://feeds.feedburner.com/timpoisot/">RSS feed for new posts</a></li>
</ul>
</div>

<div class="small-8 columns">
   <h5>Colophon</h5>
   <p>
	  This website uses <a href='http://foundation.zurb.com/'>Foundation
	  5</a>, <a href='http://fortawesome.github.io/Font-Awesome/'>Font
	  Awesome</a>, is built with <a
	  href='https://github.com/mojombo/jekyll'>Jekyll</a>, and hosted by <a
	  href='http://pages.github.com/'>GitHub pages</a>. Contents under the
	  <a href='http://creativecommons.org/licenses/by-sa/3.0/deed.en_US'>CC
	  BY-SA 3.0</a> licence.
   </p>
</div>


   </div>
</div>
</body>

</html>
